ansible_config/playbook.yml

```diff
@@ -0,0 +1,57 @@
+---
+- name: Devops
+  hosts: myhosts
+  remote_user: ec2-user
+  vars_prompt:
+
+    - name: trello_api_key
+      prompt: Enter trello api key
+    - name: trello_api_token
+      prompt: Enter trello api token
+
+  tasks:
+
+    - name: Install Git
+      become: true
+      yum:
+        name: git
+        state: installed
+    - name: Install Python 3
+      become: true
+      yum:
+        name: python3
+        state: latest
```

Comment:
Potentially a very conscious decision to use installed vs latest, but worth noting that if we do want the same state then we can combine these:
```yml
    - name: Install Git & Python3
      become: true
      yum:
        name: 
        - git
        - python3
        state: latest
```